lib/bundler/lockfile_parser.rb

```diff
@@ -195,5 +198,9 @@ def parse_bundled_with(line)
         @bundler_version = Gem::Version.create(line)
       end
     end
+
+    def parse_ruby(line)
+      @ruby_version = line.strip
```

Comment:
@segiddins Parsing is done later down the chain, it would be more work with no benefit to do it here.